vsts-gn.yml

```diff
@@ -0,0 +1,72 @@
+resources:
+- repo: self
+phases:
+- phase: Build_Electron_via_GN
+  queue:
+    timeoutInMinutes: 180
+  steps:
+  - bash: |
+      git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+      echo "##vso[task.setvariable variable=PATH]$PATH:`pwd`/depot_tools"
+    name: Setup_depot_tools
+
+  - bash: |
+      gclient config \
+        --name "src/electron" \
+        --unmanaged \
+        --cache-dir "/Users/electron/libcc_cache" \
+        "https://github.com/electron/electron"
+      gclient sync --with_branch_heads --with_tags
+    name: gclient_sync
+
+  - bash: |
+      cd src
+      export CHROMIUM_BUILDTOOLS_PATH=`pwd`/buildtools
+      export SCCACHE_WRAPPER="`pwd`/libchromiumcontent/tools/sccache/aad2120/mac/sccache"
```

Comment:
In use of `SCCACHE_WRAPPER` below, might want to wrap in quotes to bulletproof against spaces in `pwd`

Reply:
> export SCCACHE_WRAPPER="`pwd`/libchromiumcontent/tools/sccache/aad2120/mac/sccache"

Probable it would be better to use a wrapper script we have for sccache:  `script/sccache`.
It will handle version changes and possible actual binary path changes.